actionpack/lib/action_controller/metal/renderers.rb

```diff
@@ -65,19 +98,49 @@ def use_renderers(*args)
         self._renderers = renderers.freeze
       end
       alias use_renderer use_renderers
+
+      # See <tt>Renderers.use_renderers</tt>
+      def use_serializers(*args)
+        serializers = _serializers + args
+        self._serializers = serializers.freeze
+      end
+      alias use_serializer use_serializers
+    end
+
+    def has_serializer?(key)
+      _serializers.key?(key)
     end
 
+    def serializer_for(key)
+      _serializers[key]
+    end
+
+    def assert_serializer!(serializer_name)
+      return true if has_serializer?(serializer_name)
+      raise ActionController::MissingSerializer, "There is no '#{serializer_name}' serializer.\n" <<
+            "Known serializers are #{_serializers.keys}"
+    end
```

Comment:
Funny story.  [ActiveModelSerializers had an `assert_serializer` test helper that was removed, and has a PR to re-add it ready to be merged](https://github.com/rails-api/active_model_serializers/pull/1390).  I'll need to think about the differences in what each gives.  The AMS one is more concerned with the serializer class, not the rendering serializer. i.e.` assert_serializer(PostSerializer)` vs. `assert_serializer!(:json)`